src/frontend-admin/src/components/reusables/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react'
+import styles from './Dropdown.module.css'
+import { DropdownList } from './DropdownList'
+//import title to put the button inside of the corner brackets
+
+interface IDropdownProps {
+    content: {name: string, item?: any, onClick?: Function}[]
+}
+
+export const Dropdown: React.FC<IDropdownProps> = (props) => {
+    const {content} = props
+    const [selected, setSelected] = useState(content[0])
+
+    return (
+        <div className={styles.dropdownMain}>
+        
+            <div className={styles.dropdownContainer}>
+                <DropdownList 
+                    triggerElement={({isOpen, toggle}) => (
```

Comment:
Damn, getting into the fancy react now :) Render props are crazy powerful

Reply:
I'm also really impressed TS is smart enough to implicitly type the functional props here. Didn't know it would do that.